lib/wine_o_inventory_api_web/controllers/api/wine_controller.ex

```diff
@@ -7,4 +7,19 @@ defmodule WineOInventoryApiWeb.WineController do
         wines = Products.list_wines()
         render(conn, "index.json", wines: wines)
     end
+
+    def create(conn, %{"data" => wine_params}) do
+        wine_attributes = wine_params["attributes"]
+        case Products.create_wine(wine_attributes) do
+            {:ok, wine} -> 
+                conn
+                |> put_status(:created)
+                |> render("show.json", wine: wine)
```

Comment:
I was expecting something like
```suggestion
                |> render("created.json", wine: wine)
```
Probably because of my much greater familiarity with Rails, I wasn't expecting the outcome of "create" to render a "show" view.

Reply:
I was thinking that in the beginning, but `show.json` can be reused when if I ever create the functionality for getting a particular wine resource.